src/Numerics/DGMethods/DGModel.jl

```diff
@@ -14,6 +14,28 @@ struct DGModel{BL, G, NFND, NFD, GNF, AS, DS, HDS, D, DD, MD}
     diffusion_direction::DD
     modeldata::MD
 end
+
+function basic_grid_info(dg::DGModel)
+    grid = dg.grid
+    topology = grid.topology
+
+    dim = dimensionality(grid)
+    N = polynomialorder(grid)
+
+    Nq = N + 1
+    Nqk = dim == 2 ? 1 : Nq
+    Nfp = Nq * Nqk
+    Np = dofs_per_element(grid)
+
+    nelem = length(topology.elems)
+    nvertelem = topology.stacksize
+    nhorzelem = div(nelem, nvertelem)
+    nrealelem = length(topology.realelems)
+    nhorzrealelem = div(nrealelem, nvertelem)
+
+    return (Nq, Nqk, Nfp, Np, nvertelem, nhorzelem, nhorzrealelem, nrealelem)
+end
```

Comment:
@simonbyrne Is this something we want?

Reply:
Yes, something like this is useful. The batched methods use this info.

Reply:
Yes, the batched methods use this info.

Reply:
this was my rough draft at this. it could be used in a lot of the DG kernels, but I didn't want to go through and edit all of those yet. the functions that actually use this are coming in the next PR

Reply:
moved to #1268